### PR TITLE
add `Do Not Sell Data` flag

### PR DESCRIPTION
**Pull request by vrabeshko**

### CODE REVIEW

**Comment by vrabeshko** [comment]
[L - Implement CCPA "Do Not Sell" flag](https://trello.com/c/G7RIObDq/1565-l-implement-ccpa-do-not-sell-flag)

File: src/xcoobee/sdk/Consents.js [hunk 1]
```diff
@@ -633,6 +633,40 @@ class Consents {
     }
   }
 
+  /**
+    * Set or extend `Do Not Sell Data` flag
```
Comment (dhurlburtusa) [inline src/xcoobee/sdk/Consents.js]:
```suggestion
   * Set or extend `Do Not Sell Data` flag
```

File: src/xcoobee/api/ConsentsApi.js [hunk 2]
```diff
@@ -628,6 +628,36 @@ const shareConsents = (apiUrlRoot, apiAccessToken, campaignRef, campaignId = nul
...
+ * @param {string} email
+ *
+ * @returns {Promise<Object>} - The result.
+ * @property {boolean} confirmed - Flag indicating whether the change is confirmed.
+ *
+ * @throws {XcooBeeError}
+ */
+const dontSellData = (apiUrlRoot, apiAccessToken, email) => {
+  const mutation = `
+    mutation dontSellData($email: String!){
```
Comment (dhurlburtusa) [inline src/xcoobee/api/ConsentsApi.js]:
```suggestion
    mutation dontSellData($email: String!) {
```

File: src/xcoobee/api/ConsentsApi.js [hunk 3]
```diff
@@ -628,6 +628,36 @@ const shareConsents = (apiUrlRoot, apiAccessToken, campaignRef, campaignId = nul
...
+      do_not_sell_data(email: $email){
+          user_email
```
Comment (dhurlburtusa) [inline src/xcoobee/api/ConsentsApi.js]:
```suggestion
        user_email
```

File: src/xcoobee/api/ConsentsApi.js [hunk 4]
```diff
@@ -628,6 +628,36 @@ const shareConsents = (apiUrlRoot, apiAccessToken, campaignRef, campaignId = nul
...
+ *
+ * @returns {Promise<Object>} - The result.
+ * @property {boolean} confirmed - Flag indicating whether the change is confirmed.
+ *
+ * @throws {XcooBeeError}
+ */
+const dontSellData = (apiUrlRoot, apiAccessToken, email) => {
+  const mutation = `
+    mutation dontSellData($email: String!){
+      do_not_sell_data(email: $email){
```
Comment (dhurlburtusa) [inline src/xcoobee/api/ConsentsApi.js]:
```suggestion
      do_not_sell_data(email: $email) {
```

**Comment by vrabeshko** [comment]
@dhurlburtusa thanks for suggestions!

**Pull request merged**